src/helix/client/client_ext.rs

```diff
@@ -556,29 +631,38 @@ impl<'client, C: crate::HttpClient + Sync + 'client> HelixClient<'client, C> {
         Ok(resp.data.total)
     }
 
-    /// Get games by ID. Can only be at max 100 ids.
-    pub async fn get_games_by_id<T>(
+    /// Get games by ID.
+    ///
+    /// # Examples
+    ///
+    /// ```rust, no_run
+    /// # #[tokio::main]
+    /// # async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
+    /// # let client: helix::HelixClient<'static, twitch_api::client::DummyHttpClient> = helix::HelixClient::default();
+    /// # let token = twitch_oauth2::AccessToken::new("validtoken".to_string());
+    /// # let token = twitch_oauth2::UserToken::from_existing(&client, token, None, None).await?;
+    /// use twitch_api::{types, helix};
+    /// use futures::TryStreamExt;
+    ///
+    /// let games: Vec<helix::games::Game> = client.get_games_by_id(&["509658", "32982", "27471"][..].into(), &token).try_collect().await?;
```

Comment:
```suggestion
    /// let games: Vec<helix::games::Game> = client
    ///     .get_games_by_id(&["509658", "32982", "27471"][..].into(), &token).try_collect().await?;
```

Wrapping to two lines is fine here (at least on my monitor).